### ISSUE TITLE
Warp Speed dashed lines do not consider rounding

### ISSUE THREAD

**Issue by Bertie690** [sirgwain/craig-stars#391]
Oddly, the dashed lines that display your ship's position after each year of travel do not take into account the 1LY margin of error when rounding positions, so heading to a waypoint 49.9LY away at W7 will have a second dashed line start the remaining 0.9 LY away from the planet (since it only thinks you'd travel 49LY).
I'd presume a simple fix would be adding code to round down the distance used to draw the lines if the ETA to next waypoint is <=1 yr and `1 > (that yr dist - warpSpeed ^ 2) > 0`.

**Comment by Bertie690** [comment]
@sirgwain i think you fixed this in a recent release right?
(Not closinf yet cuz unsure)

**Comment by sirgwain** [comment]
Not quite. I start the line a bit away from the fleet, otherwise it looks weird, but the dashes are based on how many years it takes, so they don’t show exactly where you will end up after a year of travel. They’re a little off.

**Comment by Bertie690** [comment]
Idea: maybe we could stop the dotted line 1 year away from the target and then draw 1 final solid (ie non-dashed) line starting from the end of the dotted line's final dash straight to the target (artificially extending the final line segment a bit).
